src/definitions/modules/dropdown.js

```diff
@@ -1942,12 +1942,16 @@ $.fn.dropdown = function(parameters) {
                 $choice.find(selector.menu).remove();
                 $choice.find(selector.menuIcon).remove();
               }
-              return ($choice.data(metadata.text) !== undefined)
```

Comment:
Only two minor changes here would fix the same instead of 10 new lines.
```
return ($choice.data(metadata.text) !== undefined)
  ? $choice.data(metadata.text)
  : (preserveHTML)
    ? $choice.html() && $choice.html().trim()
    : $choice.text() && $choice.text().trim()
;
```

Reply:
Okey, no problem!